test/integration/users.test.js

```diff
@@ -1023,4 +1023,61 @@ describe("Users", function () {
         });
     });
   });
+
+  describe("POST /users/migrate", function () {
+    beforeEach(async function () {
+      superUserId = await addUser(superUser);
+      superUserAuthToken = authService.generateAuthToken({ userId: superUserId });
+    });
+
+    afterEach(async function () {
+      await cleanDb();
+    });
+    it("Should return 401 when user is unauthorize", function (done) {
+      chai
+        .request(app)
+        .post("/users/migrate")
+        .end((err, res) => {
+          if (err) {
+            return done();
+          }
+          expect(res).to.have.status(401);
+          expect(res.body.message).to.equal("Unauthenticated User");
+          return done();
+        });
+    });
+    it("Should update the user", async function () {
+      const response1 = await chai
+        .request(app)
+        .post(`/users/migrate`)
+        .set("Cookie", `${cookieName}=${superUserAuthToken}`);
+      expect(response1).to.have.status(200);
+      expect(response1.body).to.eql({
+        statusCode: 201,
+        message: `All Users github_user_id added successfully`,
+      });
+      const response2 = await chai.request(app).get(`/users`).set("cookie", `${cookieName}=${superUserAuthToken}`);
+      expect(response2).to.have.status(200);
+      response2.body.users.forEach((document) => {
+        expect(document).to.have.property(`github_user_id`);
```

Comment:
try getting the used from the db and checking if the new field got added to them

Reply:
The same thing Randhir and I have performed the route will hit the DB and Response1 for storing and Response 2 variable for checking the same. Thank You